d2cli/main.go

```diff
@@ -749,19 +757,19 @@ func renderPDF(ctx context.Context, ms *xmain.State, plugin d2plugin.Plugin, opt
 	}
 
 	for _, dl := range diagram.Layers {
-		_, err := renderPDF(ctx, ms, plugin, opts, "", page, ruler, dl, pdf, currBoardPath, pageMap)
+		_, err := renderPDF(ctx, ms, plugin, opts, "", page, ruler, dl, pdf, currBoardPath, LAYERS, pageMap)
```

Comment:
why not just modify `currBoardPath` here instead of passing along another parm

Reply:
It's not so clean.
If I do it here I'd need to extend the slice outside the loop if there are layers/scenarios/steps, and then only change the last element of the slice, otherwise I'd be concatenating sibling boards instead of descendants.
Also, it would require to move the `if diagram.Name == "" {path = root}` outside this function, requiring 2 places to keep in sync.

Reply:
can you do a commit for me to see? not convinced

Reply:
see the last commit